packages/opentelemetry-metrics/src/Utils.ts

```diff
@@ -23,5 +23,5 @@ const COMMA_SEPARATOR = ',';
  * @returns The hashed label values string.
  */
 export function hashLabelValues(labelValues: string[]): string {
-  return labelValues.sort().join(COMMA_SEPARATOR);
+  return labelValues.join(COMMA_SEPARATOR);
```

Comment:
I removed `sort()` here because per the [specs](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-metrics-user.md#label-set-calling-convention), `meter.labels(...)` function is called before calling any instrument function, thus labels passing to any instrument should be already sorted based on keys. I think we should keep `hashLabelValues` function to map values and the handle that created the certain values.

Reply:
> this may create mistake on user's end when user misuses instrument's method without calling meter.labels(...) and pass an unsorted labels directly.

This is a valid point, I am gonna give another read on specs. Let us know if you have a better solution to handle this.